src/study.js

```diff
@@ -0,0 +1,66 @@
+async function init() {
+  const elements = [...document.querySelectorAll("[data-i18n-message]")];
```

Comment:
nit: NodeList is iterable, no need for the spread here

Reply:
:heavy_check_mark: